src/QuDynamics.jl

```diff
@@ -1,6 +1,8 @@
 module QuDynamics
     using QuBase
     using Compat
+    Pkg.checkout("ODE")
```

Comment:
I guess if we include it in `REQUIRE` (and also do we add directly `Pkg.checkout("ODE")`)will the tests by travis pass ? And do we include it in `.travis.yml` by adding this 
` julia -e 'Pkg.clone("https://github.com/JuliaLang/ODE.jl.git")';` ?